SAG-Mill.md

```diff
@@ -1,2 +1,26 @@
-![](http://loenwind.info/eio/SAG_Mill.png
-)
\ No newline at end of file
+![](http://loenwind.info/eio/SAG_Mill.png)
+
+Crushes items.  Used for ore doubling and as a crafting process.
+
+The slot on the right side of the GUI is for SAG Mill Grinding Balls.  These additives increase the yield and reduce the power required for certain recipes.  The Grinding Balls are consumed, but do last for several operations.  The Main Output is a bonus to all of the possible outcomes.  The Bonus Output is applied multiplicatively to any secondary outputs.  See the example at the bottom of the page.
+
```

Comment:
Can you reword the ball text a bit? This is what I put into the recipe xml file:

```
 Grinding Balls:
  Grinding balls provide bonus outputs and reduced power use. Using the user config new grinding balls may be added and
  existing ones removed.
  
  Use of the grinding ball is restricted by the bonusType of the recipe. If it is set to "none", it will not have any grinding
  ball bonuses applied and will not reduce the durability of any balls in the mill at the time it is  processed. If it is set
  to "chance_only", the chance and power multipliers of the grinding balls are applied, but not the grinding multiplier. If it
  is set to "multiply_output" (default), all multipliers apply.
  
  - grindingMultiplier: The recipe output will be produced this many times. e.g. "3.14" would produce 3 copies of the complete
                        output set and a 14% chance for a forth copy.
  - chanceMultiplier:   The chances for the different outputs of the recipe are multiplied with this. e.g. if a recipe specifies
                        <itemStack modID="minecraft" itemName="cobblestone" chance="0.15" /> a chanceMultiplier of "3.14" would
                        make it a 47.1% chance of getting one cobblestone. Results of more than 100% do not give multiples.
  - powerMultiplier:    The "energyCost" of the recipe is multiplied with this.
  - durationRF:         This specifies how long one grinding ball will last before it is used up. The unit is the amount of
                        power used by the SAG Mill.
```

(grindingMultiplier="main" chanceMultiplier="bonus" in the GUI)

There is no "main" or "secondary" output, there only is "output with 100% chance that isn't impacted by the chanceMultiplier because of that" and "output with less than 100% chance"